src/main/java/com/epam/test_generator/entities/results/ResultTrait.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.test_generator.entities.results;
+
+import com.epam.test_generator.entities.Status;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public interface ResultTrait {
+    /**
+     * Evaluate summarized status from collection of statuses
+     *
+     * @param results collection of {@link Status}
+     * @return results evaluated by given rules
+     */
+    static Status computeStatus(List<? extends AbstractResult> results) {
```

Comment:
Let's call it `calculate`, not `compute`

Reply:
done